src/main/java/yelp/platypus/server/grpc/LuceneServer.java

```diff
@@ -17,7 +17,36 @@
  *
  */
 
-package org.apache.platypus.server.grpc;
+package yelp.platypus.server.grpc;
```

Comment:
should we have the package name be `com.yelp.nrtsearch`? We were thinking of removing references to the word platypus since its already used for a different project internally.

Reply:
Makes sense to include that here.

Reply:
Also I think the files in main should be under `com/yelp/nrtsearch` directory not just `yelp/nrtsearch`.